setup.py

```diff
@@ -63,6 +63,8 @@
     'invenio-records-rest>=1.0.0a15',
     'invenio-records-ui>=1.0.0a6',
     'invenio-files-rest>=1.0.0a3',
+    'invenio-trends-ui',
+    'invenio-trends',
```

Comment:
Who is going to publish these on PyPI? I think it would be better to publish the tag `1.0.0a1` and add the lower bound here.